### ISSUE TITLE
Show Value on Hover

### ISSUE THREAD

**Issue by Bhandejiya** [swharden/ScottPlot#115]
I can’t get to work the demo source for this. Does it require any old .dll ?

**Comment by swharden** [comment]
Hi @Bhandejiya, thanks for mentioning this. Somehow that demo got removed from the solution so its source code did not get updated as the API evolved and that demo source code no longer worked. I added that demo back to the solution and verified it works. a4bbd2a54873c801a89e79d7c1f67fe91fa45a24

Source code for this demo and implementation notes are here: https://github.com/swharden/ScottPlot/tree/master/demos/src/show-value-on-hover#highlight-point-on-hover

You can download a compiled version here: **[show value on hover.zip](https://github.com/swharden/ScottPlot/files/3537421/show.value.on.hover.zip)**

Let me know if you have any trouble! Good luck 👍 

![image](https://user-images.githubusercontent.com/4165489/63640801-d1cdaf00-c672-11e9-85f8-780376cda619.png)

**Comment by jsr1611** [comment]
> Hi @Bhandejiya, thanks for mentioning this. Somehow that demo got removed from the solution so its source code did not get updated as the API evolved and that demo source code no longer worked. I added that demo back to the solution and verified it works. [a4bbd2a](https://github.com/ScottPlot/ScottPlot/commit/a4bbd2a54873c801a89e79d7c1f67fe91fa45a24)
> 
> Source code for this demo and implementation notes are here: https://github.com/swharden/ScottPlot/tree/master/demos/src/show-value-on-hover#highlight-point-on-hover
> 
> You can download a compiled version here: **[show value on hover.zip](https://github.com/swharden/ScottPlot/files/3537421/show.value.on.hover.zip)**
> 
> Let me know if you have any trouble! Good luck 👍
> 
> ![image](https://user-images.githubusercontent.com/4165489/63640801-d1cdaf00-c672-11e9-85f8-780376cda619.png)

Dear @swharden, I want to use this feature that shows corresponding value on mouse hover in the charts, but I cannot find the documentation for it. The above link is gone somewhere. Can you help with that?
I have been using the previous version (stable). 
Using the new beta version might take some time to reorganize my project and it is still beta, I want to stick with the stable version for now.
Looking forward to hear from you.

**Comment by swharden** [comment]
Hi @Bhandejiya and @jsr1611  

I acknowledge it's been hard to follow this as the API changed between ScottPlot 4.0 to 4.1 and the documentation and code examples are moving away from the demo program and toward individual downloadable examples on the website.

Let me know if this page helps answer your questions!
https://swharden.com/scottplot/faq/mouse-position/
I'll keep working to make this more discoverable in the future

**Comment by jsr1611** [comment]
@swharden , Thank you very much for the prompt reply!
I was just about to edit my question ^^ since I found the original repo @ https://github.com/ScottPlot/ScottPlot/tree/a4bbd2a54873c801a89e79d7c1f67fe91fa45a24/demos/src/show-value-on-hover

However, I am still confused how to apply the same to **SignalPlot** (not SignalPlotXY, or SignalPlotConstXY, or ScatterPlot where we can use the given code in the repo).
Can you shed a little light on how I can do that?

**Comment by swharden** [comment]
> I found the original repo @ https://github.com/ScottPlot/ScottPlot/tree/a4bbd2a54873c801a89e79d7c1f67fe91fa45a24/demos/src/show-value-on-hover

Haha, wow, edited Aug 2019 - I probably should update or delete that - it's pretty out of date now 😅 

Regarding how to highlight points with signal plots, they have a `GetPointNearestX()` method you can use. Because of the math that allows signal plots to be so fast, they aren't capable of highlighting points depending on the vertical position of the cursor, but since signal plots are evenly spaced on the horizontal axis this is probably not an issue for most applications.
